StandardConfig/production/HighLevelReco/HighLevelReco.xml

```diff
@@ -37,6 +37,11 @@
     <parameter name="MoliereRadius_Hcal" type="float">17.19 </parameter>
   </processor>
 
+  <processor name="MyPhotonCorrectionProcessor" type="photonCorrectionProcessor">
+    <parameter name="inputCollection" type="string"> PandoraPFOs </parameter>
+    <parameter name="modifyPFOenergies"> true </parameter>
+    <parameter name="useCorrectorDefaultSet"> 1 </parameter>
```

Comment:
Even if it is a lot parameters I think we should list all parameter values here for documentation

Reply:
added all parameters (but commented out)